cloud/clouds.go

```diff
@@ -281,6 +282,16 @@ func ParseCloudMetadata(data []byte) (map[string]Cloud, error) {
 		return nil, errors.Annotate(err, "cannot unmarshal yaml cloud metadata")
 	}
 
+	if err := validateCloudSet(data); err != nil {
+
+		if _, ok := err.(*CloudValidationError); ok {
```

Comment:
If we rename the `CloudValidationError` to a `CloudValidationWarning` it is more ok to ignore this warning type from the validation function when the logging has been moved to the other package.